tests/Unit/ApparentHorizons/Test_Strahlkorper.cpp

```diff
@@ -338,7 +338,8 @@ void test_normals() {
 
   // Test surface_tangents
 
-  StrahlkorperDb::types<Frame::Inertial>::Jacobian test_surface_tangents(n_pts);
+  StrahlkorperTags::types<Frame::Inertial>::Jacobian test_surface_tangents(
```

Comment:
You should use `db::item_type<TAG>` to get the type of a tag. Please apply this approach wherever you're getting the type of a tag.